src/docs/deployments/overview/DOC.md

```diff
@@ -23,14 +23,14 @@ Below is a list of the branches on each app build repo and the environments that
 
 * prod-stable: https://cloud.redhat.com/
 * prod-beta: https://cloud.redhat.com/beta/
-* qa-stable: https://qa.cloud.paas.psi.redhat.com/
-* qa-beta: https://qa.cloud.paas.psi.redhat.com/beta
-* ci-stable: https://ci.cloud.paas.psi.redhat.com/
-* ci-beta: https://ci.cloud.paas.psi.redhat.com/beta
+* qa-stable: https://qa.cloud.redhat.com/
+* qa-beta: https://qa.cloud.redhat.com/beta
+* ci-stable: https://ci.redhat.com/
```

Comment:
this is not in the scope of the doc changes but why is ci-stable the only url that violates the pattern?

Reply:
twas a typo

Reply:
fixed!